mathics/format/mathml.py

```diff
@@ -276,7 +277,7 @@ def stylebox(self, **options) -> str:
 def graphicsbox(self, elements=None, **options) -> str:
     # FIXME: SVG is the only thing we can convert MathML into.
     # Handle other graphics formats.
-    svg_body = self.boxes_to_svg(elements, **options)
+    svg_body = boxes_to_format(self, "svg", elements=elements, **options)
```

Comment:
Why this change? I don't understand why this is done only in graphicsbox and not in other "boxes".

Reply:
The idea is to replace all the calls of the form `self.boxes_to_[format](...)` by `boxes_to_format(self, "format", ...)`.  Have I  left anyone?

Reply:
Yes:
mathics/builtin/box/graphics3d.py line 337
mathics/core/element.py line 479, 483, 487
mathics/format/mathml.py line 322

Reply:
I guess all these changes were addressed in #539. I didn't include them here to avoid making this PR longer.